src/components/pages/Stats/index.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { ReactElement, useState, useEffect } from 'react'
+
+import { Line, defaults } from 'react-chartjs-2'
+import {
+  ChartData,
+  ChartDataSets,
+  ChartOptions,
+  ChartTooltipOptions
+} from 'chart.js'
+
+import { useUserPreferences } from './../../../providers/UserPreferences'
+import { darkModeConfig } from './../../../../app.config'
+import useDarkMode from 'use-dark-mode'
+
+import styles from './index.module.css'
+import moment from 'moment'
+import fetch from 'cross-fetch'
```

Comment:
for all fetching either use `axios` or even way better use the `useSWR()` hook https://swr.vercel.app

Reply:
I thought we were going to use `cross-fetch`. I can easily switch to `useSWR` if needed

Reply:
that is only over on ocean.js. All fetching code in market uses `axios`, and when using useSWR hook it should use axios too. See the `PricesProvider` https://github.com/oceanprotocol/market/blob/main/src/providers/Prices.tsx#L46-L50

Where you can then as seen there use the same `fetchData()` utility method https://github.com/oceanprotocol/market/blob/main/src/utils/index.ts#L43-L55

Reply:
cross-fetch can be used with `useSWR`, check the graph sync pr. `axios` has CORS issues when accessing other endpoints. Not sure why stick with it

Reply:
In that case, it might be better if we update the [fetchData function](https://github.com/oceanprotocol/market/blob/main/src/utils/index.ts#L43-L55) to use cross-fetch instead of axios.